conformity/rule_settings_schema.go

```diff
@@ -71,6 +71,13 @@ func ExtraSettingSchema() *schema.Schema {
 						// region should follow the correct syntax
 					},
 				},
+				"tags": {
```

Comment:
Do we need `"ignored-regions"` added here as well?

Reply:
No need, because in last fixing we have these lines:
https://github.com/trendmicro/terraform-provider-conformity/blob/0556fec4b52f02252ccceebc9eb6f3486aeec0e6/conformity/account_settings.go#L122-L124
It can share the "regions" field, and if you check the example code, that is how it is used.
https://github.com/trendmicro/terraform-provider-conformity/blob/0556fec4b52f02252ccceebc9eb6f3486aeec0e6/example/aws/account.tf#L142-L176

We also need update the doc and add these examples:
https://registry.terraform.io/providers/trendmicro/conformity/latest/docs/resources/conformity_aws_account#argument-reference